public/components/trace_analytics/components/traces/trace_table_helpers.tsx

```diff
@@ -219,7 +219,7 @@ export const getTableColumns = (
         field: 'percentile_in_trace_group',
         name: 'Percentile in trace group',
         align: 'right',
-        sortable: true,
+        sortable: false,
```

Comment:
Question: Why is percentile not filterable?

Reply:
The percentile field is only sort-able with the currently displayed on page values as the query can not be fired off with this field. The sorting also doesn't distinguish between the trace groups. Disabling it as the sorting could be misleading as it performs differently then all the others and doesn't provide accurate information.